autoPyTorch/pipeline/components/training/trainer/cutout_utils.py

```diff
@@ -35,7 +37,12 @@ def __init__(self, patch_ratio: float,
         """
         self.use_stochastic_weight_averaging = use_stochastic_weight_averaging
         self.weighted_loss = weighted_loss
-        self.random_state = random_state
+        if random_state is None:
+            # A trainer components need a random state for
+            # sampling -- for example in MixUp training
```

Comment:
@ravinkohli Should we remove these comments, otherwise we would have to rename this cutout. But overall, I would go for removing them.